sdk/tables/azure-data-tables/README.md

```diff
@@ -7,66 +7,54 @@ The Azure Data Tables client can be used to access Azure Storage or Cosmos accou
     * an [Azure Cosmos Account][azure_cosmos_account].
 
-#### Create a storage account
-If you wish to create a new storage account, you can use [Azure Portal][azure_portal_create_account],
-[Azure PowerShell][azure_powershell_create_account], or [Azure CLI][azure_cli_create_account]:
-
-```bash
-# Create a new resource group to hold the storage account -
-# if using an existing resource group, skip this step
-az group create --name MyResourceGroup --location westus2
-# Create the storage account
-az storage account create -n MyStorageAccount -g MyResourceGroup
-```
-
-#### Creating a Cosmos DB
-If you wish to create a new cosmos storage account, you can use [Azure Cosmos DB][azure_create_cosmos].
-Create a Cosmos DB account `MyCosmosDBDatabaseAccount` in resource group `MyResourceGroup` in the subscription `MySubscription` and a table named `MyTableName` in the account.
-```bash
-az cosmosdb create --name MyCosmosDBDatabaseAccount --resource-group MyResourceGroup --subscription MySubscription
-az cosmosdb table create --name MyTableName --resource-group MyResourceGroup --acount-name MyCosmosDBDatabaseAccount
-```
-
+#### Create account
+* To create a new storage account, you can use [Azure Portal][azure_portal_create_account], [Azure PowerShell][azure_powershell_create_account], or [Azure CLI][azure_cli_create_account]:
+* To create a new cosmos storage account, you can use the [Azure CLI][azure_cli_create_cosmos] or [Azure Portal][azure_portal_create_cosmos].
 
 ### Install the package
 Install the Azure Data Tables client library for Python with [pip][pip_link]:
-
 ```bash
 pip install --pre azure-data-tables
 ```
 
-
 #### Create the client
-The Azure Data Tables client library for Python allows you to interact with two types of resources: the
-tables in your account, and the entities within the tables. Interaction with these resources starts with an
-instance of a [client](#clients). To create a client object, you will need the account's table service
-endpoint URL and a credential that allows you to access the account:
-
-```python
-from azure.data.tables import TableServiceClient
-service = TableServiceClient(account_url="https://<myaccount>.table.core.windows.net/", credential=credential)
-```
+The Azure Data Tables library allows you to interact with two types of resources:
+* the tables in your account
+* the entities within those tables.
+Interaction with these resources starts with an instance of a [client](#clients). To create a client object, you will need the account's table service endpoint URL and a credential that allows you to access the account. The account name portion of the `account_url` can be found on the page for your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by running the following Azure CLI command:
 
 ```bash
 # Get the table service URL for the account
 az storage account show -n mystorageaccount -g MyResourceGroup --query "primaryEndpoints.table"
 ```
 
```

Comment:
This client constructor snippet could probably use a one line introduction - something like:
Once you have the account URL, it can be used to create the service client